crates/connector_configs/toml/development.toml

```diff
@@ -398,26 +398,11 @@ merchant_secret="Source verification key"
   payment_method_type = "CartesBancaires"
 [[bambora.debit]]
   payment_method_type = "UnionPay"
-[[bambora.wallet]]
-  payment_method_type = "apple_pay"
-[[bambora.wallet]]
-  payment_method_type = "paypal"
 [bambora.connector_auth.BodyKey]
 api_key="Passcode"
 key1="Merchant Id"
 [bambora.connector_webhook_details]
 merchant_secret="Source verification key"
```

Comment:
Maybe we can remove this as well, if we don't support webhooks via Bambora

Reply:
Done